packages/react-dom/src/__tests__/ReactTestUtils-test.js

```diff
@@ -9,11 +9,11 @@
 
 'use strict';
 
-let createRenderer;
-let React;
-let ReactDOM;
-let ReactDOMServer;
-let ReactTestUtils;
+import ReactTestRenderer from 'react-test-renderer/shallow';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as ReactDOMServer from 'react-dom/server';
+import * as ReactTestUtils from 'react-dom/test-utils';
```

Comment:
Huh?

Reply:
What's special about this file? Why didn't you have to convert all other tests importing TestUtils or ReactDOMServer to use imports?

Reply:
OK I see now, we only do this where we import `ReactShallowRenderer`.